README.md

```diff
@@ -1,6 +1,7 @@
 # travis-kind [![Build Status](https://travis-ci.org/xmudrii/travis-kind.svg?branch=master)](https://travis-ci.org/xmudrii/travis-kind)
 
-This is an example showing how to run Kubernetes 1.11+ on [Travis-CI](https://travis-ci.org/) using [Kubernetes in Docker (`KinD`)](https://sigs.k8s.io/kind). For more details check the lightning talk I held on KubeCon North America 2018: ['Spawning Kubernetes in CI For Integration Tests'](https://sched.co/GrUv).
+This is an example showing how to run Kubernetes 1.11+ on [Travis-CI](https://travis-ci.org/) using [Kubernetes in Docker (`KinD`)](https://sigs.k8s.io/kind). For more details check the lightning talk I held on KubeCon North America 2018: ['Spawning Kubernetes in CI For Integration Tests'](https://sched.co/GrUv)
+and the [follow up article on DZone](https://dzone.com/articles/running-kubernetes-in-the-ci-pipeline-for-integrat).
```

Comment:
Can we link the article from Loodse blog https://www.loodse.com/blog/2019-03-12-running-kubernetes-in-the-ci-pipeline-/? It's the same as the article on DZone, but the blog post was originally published on Loodse blog.